src/plugins/custom_integrations/public/components/replacement_card/replacement_card.component.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+/** @jsx jsx */
+
+import { css, jsx } from '@emotion/react';
+
+import {
+  htmlIdGenerator,
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiText,
+  EuiAccordion,
+  EuiLink,
+  useEuiTheme,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { CustomIntegration } from '../../../common';
+import { usePlatformService } from '../../services';
+
+export interface Props {
+  replacements: Array<Pick<CustomIntegration, 'id' | 'uiInternalPath' | 'title'>>;
+}
+
+const URL_COMPARISON = 'https://ela.st/beats-agent-comparison';
```

Comment:
We'll need to track this tech debt, so I'd just add that we introduced this hardcoded link to "tech debt"-section in the main issue. These hardcoded links should really be using the doc-links service. We're running into a similar issue here https://github.com/elastic/kibana/pull/113666.

Reply:
What's the issue number for tech debt?  I'll add a TODO mark inline.

Reply:
actually - I take that back. We can fix that right here. This is code on the public-side. So this external doc-link system can be used here cc @lcawl

Reply:
I'm going to ship as-is and follow-up, so we don't hold back this set of changes.